mkdocs_simple_plugin/plugin.py

```diff
@@ -72,17 +74,18 @@ def gen_from_dir(self):
         for root, dirs, files in os.walk("."):
             if self.include_dir(root):
                 for f in files:
-                    if ".md" in f:
-                        doc_root = "./" + self.docs_dir + root[1:]
-                        orig = "{}/{}".format(root, f)
-                        new = "{}/{}".format(doc_root, f)
-                        try:
-                            os.makedirs(doc_root, exist_ok=True)
-                            shutil.copy(orig, new)
-                            print("{} --> {}".format(orig, new))
-                            paths.append((orig, new))
-                        except Exception as e:
-                            print("ERROR: {}.. skipping {}".format(e, orig))
+                    for extension in self.include_extensions:
+                        if extension in f:
```

Comment:
You could make this a bit more pythonic this way:

```suggestion
                   if any(extension in f for extension in self.include_extensions):
```
(ha, looks like there are other places I could apply this pattern!)